src/Admin/Block/ContainerBlockAdmin.php

```diff
@@ -38,12 +36,7 @@ protected function configureListFields(ListMapper $listMapper)
      */
     protected function configureFormFields(FormMapper $formMapper)
     {
-        $formMapper
-            ->with('form.group_general')
-                ->add('parentDocument', TreeModelType::class, array('root_node' => $this->getRootPath(), 'choice_list' => array(), 'select_root_node' => true))
-                ->add('name', TextType::class)
-            ->end()
-        ;
+        parent::configureFormFields($formMapper);
```

Comment:
if the only thing we do is call the parent, lets just remove the method completely here